src/components/sidebar-menu-item/sidebar-menu-item.stories.js

```diff
@@ -27,25 +26,25 @@ export const withTheme = () => (
   <>
     <SidebarMenuItem
       isTitle
-      theme={{ highlightColor: "#F6DD62", backgroundColor: "#F6F9FC" }}
+      style={{ "--highlightColor": "#F6DD62", "--backgroundColor": "#F6F9FC" }}
```

Comment:
Is the "--" prefix a react thing or am I missing something ?

Reply:
No, it's a CSS variable. Basically with style prop you can send any CSS. In the example above, I added to CSS variable that will change the color of the sidebar menu.

Reply:
The is a breaking change and we should have a note for projects which are using this component to update the props

Reply:
After merge I will take care on updating this on all projects that depend on fe-components.

Reply:
After merging this maybe we should also merge [this](https://github.com/code4romania/taskforce-fe-components/pull/150) and then update projects and release notes

Reply:
definitely